### ISSUE TITLE
Import Reviews from ORCid

### ISSUE THREAD

**Issue by atfornes** [bloxberg-org/peer-review-app#31]
[orcid.org](orcid.org) already registers peer reviews online, and we could import such reviews. Itallows journals and other authorized actors such [Publons](publons.com) to write those reviews, so the system could trust such reviews as verified.

See: https://support.orcid.org/hc/en-us/articles/360006971333-Peer-Review

**Comment by crossoveranx** [comment]
Agreed, we could add an ORCiD import to the Add Review page where the user provides their ORCiD to pull in all their information.
